Multitouch Support/Native/VoodooI2CNativeEngine.cpp

```diff
@@ -20,8 +20,16 @@ MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouc
     message.contact_count = event.contact_count;
     memset(message.transducers, 0, VOODOO_INPUT_MAX_TRANSDUCERS * sizeof(VoodooInputTransducer));
     
+    VoodooI2CDigitiserTransducer* transducer = (VoodooI2CDigitiserTransducer*) event.transducers->getObject(0);
+
+    if (!transducer)
+        return MultitouchReturnBreak;
+    
+    if (transducer->type == kDigitiserTransducerStylus)
+        stylus_check = 1;
+    
     for (int i = 0; i < event.contact_count; i++) {
-        VoodooI2CDigitiserTransducer* transducer = (VoodooI2CDigitiserTransducer*) event.transducers->getObject(i);
+        VoodooI2CDigitiserTransducer* transducer = (VoodooI2CDigitiserTransducer*) event.transducers->getObject(i+stylus_check);
```

Comment:
would be better to reformat the `i+stylus_check` here?

Reply:
It's not clear to me what you are suggesting.  Please elaborate or create a pull request if you feel something is incorrect.

Reply:
I meant change `i+stylus_check` to `i + stylus_check`

Reply:
Oh, I see.  lol.  Sure, if that's the convention that is being used now.  Thanks for pointing it out.

Reply:
It is just my personal suggestion. It seems we don't have a style guideline for this repo.

Reply:
We should embrace the linter we’re using for some of the project code in other parts as a start :)